index.js

```diff
@@ -0,0 +1,7 @@
+import ADBDevice from './lib/adb-device';
+
+function connectToDevice (type) {
```

Comment:
You can configure the main file in package.json and get rid of index.js if you want.